api/dataset.ts

```diff
@@ -0,0 +1,8 @@
+import { client } from "./client.ts";
+
+export const datasets = {
+  create: client("/datasets").post,
+  get: client("/datasets/{id}").get,
+  list: client("/datasets").get,
+  delete: client("/datasets/{id}").delete,
```

Comment:
Seems to match https://docs.digitalocean.com/reference/paperspace/pspace/api-reference/#tag/Machine ; just wonder if an `Update` is also needed, but not a stopper for this PR regardless of the answer.

Reply:
(maybe ordering by the supported `dtCreated` would be the least-suffering path? The order doesn't seem to be documented, so we could arguably change that in the future without it being considered a breaking change, and also arguably the date is as good as the name as a command output)